app/controllers/people_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PeopleController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
isn't this already in ApplicationController?

Reply:
Yeah -- whoops! Gotta remove that :(

Reply:
Looks like it! I can take this one out